quisby/pricing/cloud_pricing.py

```diff
@@ -138,13 +174,17 @@ def get_cloud_cpu_count(instance_name, region, cloud_type):
 
     elif cloud_type == "azure":
         return int(process_instance(instance_name, "size"))
-    
+
+    elif cloud_type == "gcp":
+        return int(process_instance(instance_name, "size"))
+
     elif cloud_type == "local":
         return 1
 
 
 
 if __name__ == "__main__":
     region = "US East 2"
-    print(get_azure_pricing("Standard_D32s_v3", region))
+    #print(get_azure_pricing("Standard_D32s_v3", region))
     # print(get_aws_cpucount("i3en.24xlarge", "US East (N. Virginia)"))
```

Comment:
Should probably delete both of these since they're just debugging aids?

Reply:
Yep